SECURITY.md

```diff
@@ -0,0 +1,31 @@
+# Security policy
+
+This document gathers security-related policies and guidelines for the codebase
+available in this repository.
+
+Authentication is a sensitive domain, and as such we designed the `@inrupt/solid-client-authn-*`
+libraries are designed with a particular attention to security. In particular, 
+we decided to apply the following rules:
```

Comment:
I would maybe use a numbered list in this context, what do you think?

Reply:
Haha it initially was, but then I didn't refer back to the numbers so figured out bullet points were enough